### ISSUE TITLE
Fails with org_id interpolated to reference a resource or data source

### ISSUE THREAD

**Issue by mikeraimondi** [terraform-google-modules/terraform-google-project-factory#80]
Tested on v0.2.1 and [master](https://github.com/terraform-google-modules/terraform-google-project-factory/tree/5eca7ea5a4280c6fbf70b03dca9e75b3e068a5b5)

Possibly related to #47; we find that passing an interpolated value to `org_id` that references a data source or resource fails with this error during `refresh`:

```
module.project.null_resource.args_missing: : invalid or unknown key: ERROR: Variable `group_name` was passed. Please provide either `org_id` or `domain` variables
```

Our use case: we store shared state, like the organization and folders, in a separate backend which we reference using `terraform_remote_state`. This bug makes it impossible to reference shared state in this way.

Is there a workaround we can pursue in the near term?

**Comment by aaron-lane** [comment]
Hi @mikeraimondi.

I apologize for the delay in responding. v1.0.0 was released recently. Are you able to test the upgraded version to see if your problem is addressed?

**Comment by mikeraimondi** [comment]
Yes, this issue appears to be resolved in 1.0.0. Thanks!
